client/src/SearchBar.jsx

```diff
@@ -7,25 +7,42 @@ const SearchBar = props => (
       id="location"
       type="text"
       placeholder="location"
+<<<<<<< HEAD
```

Comment:
Seeing git diff text injected.  Anyone else see it?  Needs cleanup?

Reply:
fixed it
look again

Reply:
He changed them